packages/workspace/src/core/project-graph/project-graph.ts

```diff
@@ -41,6 +41,14 @@ import {
 } from '../nx-deps/nx-deps-cache';
 import { performance } from 'perf_hooks';
 
+export async function createProjectGraphAsync(): Promise<ProjectGraph> {
+  return createProjectGraph();
```

Comment:
(In follow up phases this will naturally not simply be a wrapper around the current sync utility)